### PR TITLE
Us55 borrower verified landing page

### PR DESCRIPTION
**Pull request by danielmarker**
New page setup and is working the same as the prototype - with the button going to the /searchdeed function. Please comment

### CODE REVIEW

**Comment by Kirosoft** [comment]
I think we should keep the image assets outside of the government uk assets directory. I would suggest /application/static/landregistry/digitalmortgage/images (if the images are specific to our project)

File: application/borrower/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+__author__ = 'danmarker'
```
Comment (endamccormack) [inline application/borrower/__init__.py]: Probably worth removing this. Your git history is your claim to fame.

File: application/templates/landing.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
+{% extends "base/lr_base_layout.html" %}
+
+{% block content %}
+<main id="content" role="main">
+
+	<h1 class="heading-large">
+		<!-- <span class="heading-secondary">Sign your mortgage deed</span> -->
```
Comment (endamccormack) [inline application/templates/landing.html]: Comment left in

**Comment by danielmarker** [comment]
Updated as per comments

**Comment by endamccormack** [comment]
Your circular number assets have their edges cut off and they're clearly scaled.

**Comment by endamccormack** [comment]
this is the landing page for the borrower_frontend but I dont think its the actual landing page is it? The landing page in my mind one be like this one where you declare what the service is https://www.gov.uk/register-to-vote

**Comment by endamccormack** [comment]
It just occurred to me that you added 26 icons when there should only be 6 from what I can see.

**Comment by michaeldfallen** [comment]
How come you are using images instead of CSS for those (what I assume are) step indicators?

**Comment by danielmarker** [comment]
@michaeldfallen I have found them from the prototype I was asked to base on - but this could be done in CSS?
@endamccormack Again the prototype had the whole folder for later use

**Comment by michaeldfallen** [comment]
@danielmarker you've got two options, both better than directly embedding images. Either put a background on a span or use css to draw the cirlce. Something like:

```
<span class="step-bullet-image">1</span>
<span class="step-bullet-css">2</span>
```

```
.step-bullet-image {
  width: 1em;
  height: 1em;
  background-image: url('/assets/my-circle-background.png');
  font-weight: bold;
  color: #ffffff;
}

.step-bullet-image {
  width: 1em;
  height: 1em;
  display: inline-block;
  font-weight: bold;
  color: #ffffff;
  background-color: #000000;
  -webkit-border-radius: 0.5em;
  -moz-border-radius: 0.5em;
  border-radius: 0.5em;
}
```

Each has it's ups and downs. Images don't scale well, and images with text especially don't since Retina displays doing the weird double pixel thing is a thing now, but css isn't always portable and can produce weird results. The css should work on [browsers back to 2007 or so](http://caniuse.com/#search=border-radius).

Here's a [jsFiddle showing this in action](https://jsfiddle.net/4yuuhrd9/)

**Comment by endamccormack** [comment]
:+1:

**Pull request merged**